node-uuid/node-uuid-global.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for node-uuid.js
+// Project: https://github.com/broofa/node-uuid
+// Definitions by: Jeff May <https://github.com/jeffmay>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/// <reference path="./node-uuid-base.d.ts" />
+
+/** Expose as global variable */
+declare var uuid: __NodeUUID.UUID;
```

Comment:
It's a pattern which I'm starting to see more and more often with definitions for node modules that don't create globals, for instance React or MaterialUI. It differentiates the TS namespace from any real global variable with the same name. I don't like underscore variables either, but I can always alias it: ```type UUID = __NodeUUID;```, which at least makes the intention explicit and eliminates a source of confusion. It's also makes it impossible to use the global variable by accident in case your definitions has both a module export and a global variable (see angular.d.ts);